src/index.tsx

```diff
@@ -10,8 +10,9 @@ import { ROLES } from './Roles';
  * @param {createjs.Stage} stage - createjs stage that has been registered for accessibility
  * @returns {Object} style object for positioning associated DOM elements
  */
-function calcDomStylesFromStage(stage) {
-  // true to return style object for putting the DOM element adjacent to the canvas (useful for debugging),
+function calcDomStylesFromStage(stage): { [key: string]: React.CSSProperties } {
```

Comment:
Can we define this type above, so that it can be used in line#67 as well without duplication...